config/autoload/global.php

```diff
@@ -121,7 +122,7 @@
             /*
              * Max. width and height which a thumbnail may have.
              */
-            'width' => 960,
+            'width' => 1920,
             'height' => 1920
```

Comment:
Maybe decrease the max height a bit? Currently you often see portrait pictures which do not fit on a single page, since they are too high.

Reply:
The new viewer actually only uses the width and dynamically determines the max height based on the aspect ratio / orientation. I just left this here until we fully remove the old viewer.

We should reconsider these dimensions sometime in the future however, probably at the same time when we think about larger displays.